t/Util.pm

```diff
@@ -525,6 +555,88 @@ sub spawn_h2_server {
     return $server;
 }
 
+# usage: see t/90h2olog.t
+package H2ologTracer {
+    use POSIX ":sys_wait_h";
+
+    sub new {
+        my ($class, $opts) = @_;
+        my $h2o_pid = $opts->{pid} or Carp::croak("Missing pid in the opts");
+        my $h2olog_args = $opts->{args} // [];
+        my $h2olog_prog = t::Util::bindir() . "/h2olog";
+
+        my $tempdir = File::Temp::tempdir(CLEANUP => 1);
+        my $output_file = "$tempdir/h2olog.jsonl";
```

Comment:
Just curious, but what does "l" of "jsonl" stand for?

Reply:
That's because the output format is [JSON Lines](https://jsonlines.org/), and the spec suggests the `.jsonl` extension.

Reply:
Thanks!